src/css/controls/imports/idle-variants.less

```diff
@@ -0,0 +1,40 @@
+@import "../../shared-imports/vars";
+
+.jw-state-idle {
+    .jw-icon-display[class*="jw-ab-idle"] {
```

Comment:
Why are we checking for a class attribute that contains "jw-ab-idle"? It would be simpler and clearer to list the different classes here.

Reply:
Where is the "stroke" or "fill" applied?

Reply:
These properties are applied to all variants, and it would be redundant to duplicate them for each variant.